spark-cassandra-connector/src/test/java/com/datastax/spark/connector/mapper/JavaTestBean.java

```diff
@@ -0,0 +1,43 @@
+package com.datastax.spark.connector.mapper;
+
+import com.datastax.driver.mapping.annotations.Column;
+
+import java.io.Serializable;
+
+/**
+ * This is a Java Bean style class with Java Driver style annotations built in.
+ * This class also contains nested UDTs with its own mappings
+ */
+public class JavaTestBean implements Serializable {
+
+    @Column(name = "property_1")
```

Comment:
I know this is unnecessary but touched we make these a little more different. property1 -> "cassandraproperty1" 
Just so the mapping is much clearer when we look back :)